Cliques.js

```diff
@@ -38,14 +38,33 @@ Clique.prototype.add = function(ws) {
 	ws.on('close', (m) => {
 		this.remove(ws)
 	})
+
 }
 
 Clique.prototype.onMessage = function(sender, message) {
 	var payload = {}
 	try {
 		payload = JSON.parse(message)
-		payload.id = sender.id
-		payload.name = sender.name
+		switch (payload.event) {
+			case 'friend-joined':
+				//Chris, how can I clean this up? This looks
```

Comment:
That's easy! There's no need to check friends.length. `[].map(prettyMuchAnything)` returns [].